source/classes.tex

```diff
@@ -6660,53 +6667,142 @@
 A defaulted comparison operator function\iref{over.binary}
 for some class \tcode{C}
 shall be a non-template function
-declared in the \grammarterm{member-specification} of \tcode{C}
+defined in the \grammarterm{member-specification} of \tcode{C}
 that is
 \begin{itemize}
 \item
-a non-static const member of \tcode{C} having
-one parameter of type \tcode{const C\&}, or
+a non-static const non-volatile member of \tcode{C} having
+one parameter of type \tcode{const C\&} and either
+no \grammarterm{ref-qualifier} or
+the \grammarterm{ref-qualifier} \tcode{\&}, or
 
 \item
 a friend of \tcode{C} having either
 two parameters of type \tcode{const C\&} or
 two parameters of type \tcode{C}.
 \end{itemize}
-
-\pnum
-A defaulted comparison operator function for class \tcode{C}
+A defaulted comparison operator function for class \tcode{C} that
+is not defined as deleted is
+\indextext{operator!comparison!defaulted!implicitly defined}%
```

Comment:
Please check the existing index entries and find a suitable place. This one isn't.
Maybe "operator function!comparison!implicitly-defined".

Reply:
For all my index entries, I do scan the existing ones and to try and find a suitable place.  As I said in my "Notes/issues", it isn't clear how we should write index entries.  I don't like having so many levels either, but which part to leave out, and which parts to combine?

For this case,  we don't have "operator function" but we do have "operator", so I'll just remove the "defaulted" level.  Thanks!